apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -8,47 +8,35 @@ import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
 
 export const COLUMNS = {
   QUESTION: 0,
-  ANSWER_1: 1,
-  ANSWER_2: 2,
-  NOT_ANSWERED: 3,
 };
 
-const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionOne}
-        isCorrectAnswer={rowData.optionOneIsCorrect}
-      />
-  );
-};
+const alphabetMapper =  [
+                          commonMsg.answerOptionA(),
+                          commonMsg.answerOptionB(),
+                          commonMsg.answerOptionC(),
+                          commonMsg.answerOptionD(),
+                          commonMsg.answerOptionE(),
+                          commonMsg.answerOptionF(),
+                          commonMsg.answerOptionG(),
+                        ];
 
-const answerOptionsTwoFormatter = (percentAnsweredOptionOne, {rowData}) => {
+const answerColumnsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex, property}, index) => {
+  const column = (columnIndex - 1);
+  const cell = rowData.answers[column];
   return (
       <MultipleChoiceAnswerCell
         id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionTwo}
-        isCorrectAnswer={rowData.optionTwoIsCorrect}
-      />
-  );
-};
-
-const notAnsweredFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.notAnswered}
+        percentValue={cell && `${cell.percentAnswered}%` || '-'}
```

Comment:
Nit: Instead of using these `cell &&`'s, it might be clearer to separate out the missing data case:
```
if (!cell) {
  return (<MultipleChoiceAnswerCell id={...} percentValue='-' isCorrectAnswer={false} />;
}
return (
  <MultipleChoiceAnswerCell
    id={...}
    percentValue={`${cell.percentAnswered}%`}
    isCorrectAnswer={cell.isCorrectAnswer}
  />
);
```
I could go either way though, do what you think looks nicer.

Reply:
Thanks!